pkg/specgenutil/createparse.go

```diff
@@ -1,13 +1,14 @@
-package common
+package specgenutil
 
 import (
 	"github.com/containers/common/pkg/config"
+	"github.com/containers/podman/v3/pkg/domain/entities"
 	"github.com/pkg/errors"
 )
 
 // validate determines if the flags and values given by the user are valid. things checked
 // by validate must not need any state information on the flag (i.e. changed)
-func (c *ContainerCLIOpts) validate() error {
+func validate(c *entities.ContainerCLIOpts) error {
```

Comment:
Why the change here?

Reply:
since `PodCreateOptions` (which is already in entities) and `ContainerCLIOpts` are now used and called in basically all of the same places, the import cycle does not allow `common.` in many places. Makes sense since ContainerCLIOpts is more widespread now.